src/main/java/uk/gov/pay/adminusers/queue/event/EventMessageHandler.java

```diff
@@ -90,7 +90,7 @@ public void processMessages() throws QueueException {
                         handleDisputeWonMessage(message.getEvent());
                         break;
                     default:
-                        logger.warn("Unknown event type: {}", eventType);
+                        logger.warn("Unknown event type: {}", message.getEvent().getEventType());
```

Comment:
Should we stop logging at warn level here? DISPUTE_INCLUDED_IN_PAYOUT events will reach adminusers but adminusers will never care about them.

Reply:
yeah, don't need to be at the warning level. Updated to `info`.